scvi/model/base/_base_model.py

```diff
@@ -272,7 +272,11 @@ def train(
         task_kwargs = vae_task_kwargs if isinstance(vae_task_kwargs, dict) else dict()
         self._pl_task = task_class(self.model, len(self.train_indices_), **task_kwargs)
 
-        self.trainer.fit(self._pl_task, train_dl, val_dl)
+        if train_size==1.0:
+            # circumvent the empty DL problem
+            self.trainer.fit(self._pl_task, train_dl, train_dl)
```

Comment:
I think what you can do is just pass the train_dl and that's it, no val loop this way.